Marlin/ubl_G29.cpp

```diff
@@ -448,7 +448,7 @@
             y_pos = current_position[Y_AXIS];
           }
 
-          float height = code_seen('H') && code_has_value() ? code_value_float() : Z_CLEARANCE_BETWEEN_PROBES;
+          float height = Z_CLEARANCE_BETWEEN_PROBES;
 
           if (code_seen('B')) {
             card_thickness = code_has_value() ? code_value_float() : measure_business_card_thickness(height);
```

Comment:
@bgort Since the `height` variable is used here by `measure_business_card_thickness`, shouldn't the handling of `H` come ahead of this line?